src/internal/CircleRipple.js

```diff
@@ -16,6 +32,7 @@ class CircleRipple extends Component {
   static defaultProps = {
     opacity: 0.1,
     aborted: false,
+    onRippleStart: () => {},
```

Comment:
We have a mixed approach on the code base. Sometimes we use default properties and sometimes we check the existence of the property.
On the `next` branch, we follow the check existence solution. Can we do the same here?